assignments/objects.js

```diff
@@ -50,10 +50,18 @@ console.log(antonietta.gender);
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
-
+function speak (name) {
+  return `Hello my name is ${this.name}`
+}
+kennan.speak = speak;
+console.log(kennan.speak());
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
 //console.log(antonietta.multiplyNums(3,4));
-
+ 
+antonietta.multiplyNums = function(x, y) {
```

Comment:
Remember you could always shorten this out by declaring it into one word. Good job though.